apps/ui-mobile/src/app/App.tsx

```diff
@@ -20,6 +22,14 @@ console.log(instanceToPlain(new ActivityTrackingMeta()));
 
 const App: React.FC<{}> = () => {
 	React.useEffect(() => {
+		MetaWearModule.setupPreviouslyConnectedMetaWear()
+			.then(() => {
+				showNotification('Przywrócono połączenie z urządzeniem MetaWear!');
+				MetaWearModule.blinkBlueLED(1 + 1);
+			})
+			.catch(() => {
+				showNotification('Niw wykryto urządzenia MetaWear, połącz się ponownie!');
+			});
```

Comment:
Wszystko w apce jest po angielsku. Chyba trzeba zmienić

Reply:
To ja to zmerguje i poprawie ze swoim commitem